### PR TITLE
Attribute selection styling

### PR DESCRIPTION
**Pull request by eireland**
Styles attribute buttons.
When user selects All, All button is highlighted, and all attributes are added to state.
When users selects an attribute, attribute button is highlighted, and added to state. If All was selected, all attributes are removed from state, and All button is de-selected.

### CODE REVIEW

File: src/components/attribute-selector.tsx [hunk 1]
```diff
@@ -1,39 +1,74 @@
...
+  const handleAttributeSelect = (e: React.MouseEvent<HTMLDivElement>) => {
+    const attrSelected = e.currentTarget.textContent;
+    if (allSelected) {
+      setAllSelected(false);
+    }
+    setState(draft => {
+      if (allSelected) {
+        draft.attributes = [];
+      }
+      attrSelected && draft.attributes.push(attrSelected);
```
Comment (dougmartin) [inline src/components/attribute-selector.tsx]: It seems like this doesn't handle toggling the attributes but rather just keeps adding them to the list.

**Review by dougmartin — COMMENTED** [review]
I have a question about how the attribute toggling works.  See inline.

File: src/components/attribute-selector.tsx (hunk 1 above)
Comment (lublagg) [reply src/components/attribute-selector.tsx]: Same comment as Doug - it seems we should check if the clicked attribute is already selected, and de-select it if so.

**Review by lublagg — CHANGES_REQUESTED** [review]

**Review by lublagg — APPROVED** [review]
LGTM 👍

**Pull request merged**